_posts/2025-08-04-correo.ejecutivo-matrizderiesgo.md

```
@@ -0,0 +1,22 @@
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [Riesgo, Ejecutivo, Cliente, Línea de crédito]
+comments: false
+author: Catalina Pinilla
+---
+
+## Matriz de riesgo - Ejecutivo.
+
+En el apartado de línea de crédito en la `Matriz de riesgo` se ha agregado el mail del ejecutivo a quien pertenece el cliente, para ello debes ingresar el nombre o el rut del cliente que quieras consultar y aparecerán todos los datos de la Línea de crédito aprobada del cliente.
+
+#### Aquí te mostramos como se visualiza esta nueva implementación.
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)